test/e2e/policies.test-suite/balloons/n4c16/test22-isolcpus/code.var.sh

```diff
@@ -0,0 +1,49 @@
+helm-terminate
+helm_config=${TEST_DIR}/balloons-isolcpus.cfg helm-launch balloons
+
+vm-command "grep isolcpus=0,1 /proc/cmdline" || {
+    if [[ "$distro" == *fedora* ]]; then
+        fedora-set-kernel-cmdline "isolcpus=0,1"
+    else
+        ubuntu-set-kernel-cmdline "isolcpus=0,1"
+    fi
+    vm-reboot
+    sleep 5
```

Comment:
This will be unreliable. It would be better to have something like

```
reboot-node() {
    vm-reboot
    sleep 5
    timeout=120 host-wait-vm-ssh-server $OUTPUT_DIR
}
```

and then call `reboot-node` to reboot the node and wait for it to come up and be ready for ssh commands.